jnosql-mongodb/src/main/java/org/eclipse/jnosql/databases/mongodb/communication/MongoDBUtils.java

```diff
@@ -74,7 +74,7 @@ public static List<org.eclipse.jnosql.communication.document.Document> of(Map<St
         return values.keySet().stream().filter(isNotNull).map(documentMap).collect(Collectors.toList());
     }
 
-    private static org.eclipse.jnosql.communication.document.Document getDocument(String key, Object value) {
+    public static org.eclipse.jnosql.communication.document.Document getDocument(String key, Object value) {
```

Comment:
Can it be default package?

Reply:
oops sorry. this should be kept private.
I tried to make aggregate generic and left this accidently...